lib/gui/app/components/wifi/controllers/wifi.js

```diff
@@ -0,0 +1,32 @@
+
+/*
+ * Copyright 2016 resin.io
```

Comment:
2016?? :wink: 
And some of the other files added in this PR seem to be missing the appropriate copyright header?